CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+
+## 2021.7.0
```

Comment:
@xirc [IMO] I think it would be helpful for users to be able to see the diff like in other repositories (e.g. [lerna-app-library/CHANGELOG.md](https://github.com/lerna-stack/lerna-app-library/blob/6b18c4c98f301537e7272130c800c5d0e2a79dcb/CHANGELOG.md)), so that they can check the details.

Reply:
@negokaz 
I've made a PR request.
[CHANGELOG に リンクを追加する by xirc · Pull Request #48 · lerna-stack/lerna.g8](https://github.com/lerna-stack/lerna.g8/pull/48)